dot/core/service_integration_test.go

```diff
@@ -504,7 +385,11 @@ func TestMaintainTransactionPool_EmptyBlock(t *testing.T) {
 		},
 	}
 
-	ts := state.NewTransactionState()
+	ctrl := gomock.NewController(t)
+	telemetryMock := NewMockClient(ctrl)
+	telemetryMock.EXPECT().SendMessage(gomock.Any()).AnyTimes()
```

Comment:
Can we expect particular messages or are there too many being sent out?

Reply:
Um i am not sure I did not do telemetry tests, these are same as whats in dev. I had to manually bring them over though because of file change. But the contents of this file should (unless I messed it up somewhere) match service_test.go on dev branch

Reply:
Oh yeah no worry then! Sorry I'm just brute-reviewing all this 😄